### PR TITLE
Clean up outdated todo marker

### PR DESCRIPTION
**Pull request by SpacePossum**
with https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3185 merged this TODO marker is out dated

### CODE REVIEW

File: tests/Linter/ProcessLinterProcessBuilderTest.php [hunk 1]
```diff
@@ -35,7 +35,6 @@
      */
     public function testPrepareCommandOnPhpOnLinuxOrMac($executable, $file, $expected)
     {
-        // @TODO drop condition at 2.4 (as 2.4 does not support HHVM)
```
Comment (SpacePossum) [inline tests/Linter/ProcessLinterProcessBuilderTest.php]: this is done on 2.4

File: tests/Linter/ProcessLinterProcessBuilderTest.php [hunk 2]
```diff
@@ -67,9 +66,6 @@ public function testPrepareCommandOnPhpOnWindows($executable, $file, $expected)
         );
     }
 
-    /**
-     * @TODO drop condition at 2.4 (as 2.4 does not support HHVM)
-     */
```
Comment (SpacePossum) [inline tests/Linter/ProcessLinterProcessBuilderTest.php]: this is done on 2.4 as well

File: tests/Fixer/Whitespace/IndentationTypeFixerTest.php [hunk 3]
```diff
@@ -319,7 +319,7 @@ public function provideMessyWhitespacesReversedCases()
         $filteredCases = array();
         $cases = $this->provideMessyWhitespacesCases();
 
-        foreach ($cases as $key => $case) { // TODO on 5.6 bump use array_filter with ARRAY_FILTER_USE_KEY
```
Comment (SpacePossum) [inline tests/Fixer/Whitespace/IndentationTypeFixerTest.php]: done on master/2.8

**Comment by keradus** [comment]
:-1:

the TODOs are valid for 2.2 line and when they were solved, eg on 2.4, TODO was removed on 2.4

**Comment by keradus** [comment]
like if we have 1.13 with TODOs for 2.0, we didn't drop TODOs on 1.13, even if they are solved on 2.0

**Comment by SpacePossum** [comment]
fair enough, good ti know the reasoning :)

**Pull request closed without merging**